test/integration/storage/test_objects.rb

```diff
@@ -0,0 +1,115 @@
+require "helpers/integration_test_helper"
+
+def before_run
+  @connection = Fog::Google::StorageJSON.new
+  @connection.put_bucket("fog-smoke-test", options = { "predefinedAcl" => "publicReadWrite" })
+rescue Exception => e
```

Comment:
Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`?<br>Useless assignment to variable - `e`.